main.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+/*Joe bought 1k stocks for 45.50 , Broker got 2%
+ * Joe sold 1k stocks for 56.90 , Broker got 2%
+ * Output :
+ * Money paid
+ * commission paid
+ * Amount sold for
+ * commission paid
+ * Profit or Loss
+ * Constants - SHARE_BOUGHT , SHARE_SOLD , COMMISSION_RATE , PURCHASE_PRICE , SALE_PRICE
+*/
+
+int SHARE_BOUGHT = 1000 , SHARE_SOLD = 1000 ;
+double COMMISSION_RATE = 0.02, PURCHASE_PRICE = 45.5, SALE_PRICE = 56.9;
+int main() {
+    double sale_bought , sale_sold , sale_profit , commssion_paid_buy , commssion_paid_sale;
+    char new_line = '\n';
+
+    sale_bought = SHARE_BOUGHT * PURCHASE_PRICE;
```

Comment:
I recommend that you declare and initialize on the same line. But this is also rather easy to read that said when extracting variables from logic later you will see how this aforementioned technique will make your life simpler